updated getting started tutorials

cdzialo opened the pull request:
-Modified all lessons within 'A_Getting_started' tutorial and corresonding images to adhere to new Human Standing template model and example.

cdzialo committed:
-Modified all lessons within 'A_Getting_started' tutorial and corresponding images to adhere to new Human Standing template model and example.

cdzialo committed:
Merge remote-tracking branch 'remotes/origin/master' into update_tutorial

melund reviewed (commented):
A_Getting_started/_static/lesson2/image1.png seems very different from the original. Is that intended?

melund commented on A_Getting_started/lesson1.rst:
```diff
@@ -1,11 +1,11 @@
 
-The standing model can be found in the AMMR folder under
-Applications/Examples. The main file is called StandingModel.Main.any.
+The Human Standing model can be found in the AMMR folder under
```
Yes. Just revert the "Human Standing" thing.

cdzialo replied:
Done. Just committed new files.

melund commented:
Thanks a lot @cdzialo. This is exactly the update we want. Also nice that you fixed some the bad language :)

cdzialo committed:
-reverted 'Human Standing Model text to Standing Model'
-adjusted minor text errors in A_Getting_started_AMMR/index.rst and A_Getting_started_anyscript/index.rst

melund commented:
Thanks a lot @cdzialo
